### PR TITLE
Withdraw command on domains

### PR DESCRIPTION
**Pull request by rasmussoren**

### CODE REVIEW

File: lib/Net/EPP/Frame/Command/Withdraw/Domain.pm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	my ($package, $type) = @_;
+	my $self = $package->SUPER::new;
+    my $extelm = $self->createElement('extension');
+    $self->getNode('epp')->addChild($extelm);
+    my $command = $self->getNode('command');
+    $command->setNamespace( commandNamespace );
+    $command->unbindNode;
+    $extelm->addChild($command);
+    $self->getNode('transfer');
+    #->setNodeName('withdraw');
```
Comment (jonasbn) [inline lib/Net/EPP/Frame/Command/Withdraw/Domain.pm:22]: Hi @rasmussoren 

I do not completely understand why

- Line 21 is referring `transfer`
- Line 22 is commented out

Please make a comment in the code if this is a necessary hack

**Review by jonasbn — CHANGES_REQUESTED** [review]
Please see comment on lines 21 and 22

File: lib/Net/EPP/Frame/Command/Withdraw/Domain.pm (hunk 1 above)
Comment (rasmussoren) [reply lib/Net/EPP/Frame/Command/Withdraw/Domain.pm:22]: No the two line are not neccessary and can be excluded

Comment (jonasbn) [reply lib/Net/EPP/Frame/Command/Withdraw/Domain.pm:22]: You mean line 21 can also be commented out (deleted) ?

Comment (rasmussoren) [reply lib/Net/EPP/Frame/Command/Withdraw/Domain.pm:22]: Yes. I have tested without and it works as intended

**Review by jonasbn — APPROVED** [review]
Here we go

**Pull request merged**